crowdloans/rewards/polkadot.json

```diff
@@ -1,4 +1,79 @@
 {
+  "2026": {
+    "name": "Nodle",
```

Comment:
Can we remove the name, home, token, logo into projects JSON? For this json, we only want to keep the rewards information

https://github.com/parallel-finance/auction-toolkit/blob/main/crowdloans/projects/polkadot.json

Reply:
added in : 689d057